### PR TITLE
Multimodality/feat image weighting decay

### PR DESCRIPTION
**Pull request by RaphaelKreft**
### Important for Apertus 1p5 pretrain
- Add image weight decay optiom
- Fix off by one image weighting (to be verified)

### Other changes
- Add fast dataset split script
- Fix weighted dset creation script: get dset size was reading version not num sequences

### CODE REVIEW

**Comment by RaphaelKreft** [comment]
@Alvorecer721 new changes look good to me. Also like the clean integration of Audio+Image
